WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -0,0 +1,243 @@
+package org.wordpress.android.ui.people;
+
+import android.app.AlertDialog;
+import android.app.FragmentManager;
+import android.app.FragmentTransaction;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+
+import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Blog;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.ui.ActivityLauncher;
+import org.wordpress.android.ui.accounts.BlogUtils;
+import org.wordpress.android.ui.people.utils.PeopleUtils;
+import org.wordpress.android.util.ToastUtils;
+
+import java.util.List;
+
+public class PeopleManagementActivity extends AppCompatActivity
+        implements PeopleListFragment.OnPersonSelectedListener, RoleChangeDialogFragment.OnChangeListener {
+    private static final String KEY_PEOPLE_LIST_FRAGMENT = "people-list-fragment";
+    private static final String KEY_PERSON_DETAIL_FRAGMENT = "person-detail-fragment";
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.people_management_activity);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setTitle(R.string.people);
+            actionBar.setHomeButtonEnabled(true);
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        int localBlogId = BlogUtils.getBlogLocalId(WordPress.getCurrentBlog());
+        Blog blog = WordPress.getBlog(localBlogId);
```

Comment:
This seems odd since it's getting the ID of the current blog in order to lookup the current blog. And the code below doesn't deal very well with the current blog being null - most likely the activity should exit in that situation.

How about this instead?
```
Blog blog = WordPress.getCurrentBlog();
if (blog == null) {
    ToastUtils.showToast(this, R.string.blog_not_found);
    finish();
    return;
}

if (savedInstanceState == null) {
    PeopleListFragment peopleListFragment = PeopleListFragment.newInstance(blog.getLocalTableBlogId());

    getFragmentManager().beginTransaction()
            .add(R.id.fragment_container, peopleListFragment, KEY_PEOPLE_LIST_FRAGMENT)
            .commit();
}

refreshUsersList(blog.getDotComBlogId(), blog.getLocalTableBlogId());
```